soc/arm/atmel_sam0/common/gmac_fixup_samd5x.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2020 Stephanos Ioannidis <root@stephanos.io>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
Can you add a comment about what these fixups are for/why they are needed.

Reply:
I will add the explanation in the commit message as in-line comment here.

Reply:
Done